Microsoft.Toolkit.Uwp/Helpers/ConnectionHelper.cs

```diff
@@ -49,5 +49,24 @@ public static bool IsInternetAvailable
                 return NetworkInformation.GetInternetConnectionProfile() != null;
             }
         }
+        
+        /// <summary>
+        /// Gets a value indicating whether internet is established over Wi-Fi or not.
+        /// </summary>
+        /// <returns>True if internet connection is established over Wi-Fi.</returns>
+        public static bool IsInternetConnectionEstablishedOverWifi
```

Comment:
Microsoft Naming Alert!! ;)

Could this not just be something simpler like: IsWifiConnection?